cmd/snap-bootstrap/initramfs_systemd_mount.go

```diff
@@ -149,6 +156,22 @@ func doSystemdMountImpl(what, where string, opts *systemdMountOptions) error {
 	if opts.Private {
 		options = append(options, "private")
 	}
+	if len(opts.VerityHashDevice) > 0 && len(opts.VerityRootHash) == 0 {
```

Comment:
I'd say that in general we then to use `=/!=` for strings instead of len(), I think it would make this easier to read.

Reply:
Thanks, fixed.